vllm/v1/core/scheduler.py

```diff
@@ -112,6 +112,7 @@ def schedule(self) -> "SchedulerOutput":
         scheduled_resumed_reqs: List[Request] = []
         scheduled_running_reqs: List[Request] = []
         preempted_reqs: List[Request] = []
+        guided_decoding_request_ids: Dict[str, int] = {}
```

Comment:
Add a comment explaining what this is for (what is the key, what is the value, why?)

Reply:
resolved